LICENSE

```diff
@@ -1,7 +1,5 @@
 
-                                 Apache License
-                           Version 2.0, January 2004
-                        http://www.apache.org/licenses/
+Copyright (c) 2024, Network to Code, LLC
```

Comment:
Seems like a weird change to remove the "Apache License" from this file. Is this from drift manager?

Reply:
yes, I think that all this changes are coming from drift manager